lightly/api/api_workflow_upload_dataset.py

```diff
@@ -35,15 +35,25 @@
 from lightly.openapi_generated.swagger_client.rest import ApiException
 
 
+try:
+    from lightly.data import LightlyDataset
+    _lightly_dataset_available = True
+except (
+    OSError,              # Different CUDA versions for torch and torchvision
+    ModuleNotFoundError,  # No installation of torch or torchvision
```

Comment:
Why could you remove the importerror?

Reply:
Ah I removed the wrong one, good catch! `ModuleNotFoundError` is a subclass of `ImportError`. I will change it.

Reply:
Can we also update the exception catching in `lightly/__init__.py`?

```
    try:
        import torchvision.models.vision_transformer
        _torchvision_vit_available = True
    except (
        ImportError,          # Old version of torchvision
        OSError,              # Different CUDA versions for torch and torchvision
        ModuleNotFoundError,  # No installation of torchvision
    ):
```